update.sh

```diff
@@ -28,7 +28,18 @@ elif [ -x "$(command -v python)" ]; then
 						if [[ -z "$VIRTUAL_ENV" ]]; then
     							echo "warning: No VIRTUAL_ENV set. exiting."
 						else
-							python -m pip install -r requirements.txt --force-reinstall
+							REQS_INSTALLED=0
+							if [ -e /dev/kfd ]; then
+								python -m pip install -r requirements-global.txt -r requirements-rocm.txt --force-reinstall
+								REQS_INSTALLED=1
+							fi
+							if [ -x "$(command -v nvidia-smi)" ]; then
```

Comment:
I don't really like this way of auto detecting the device type. nvidia-smi might not be installed even on

Reply:
`nvcc` ships with the driver, done in https://github.com/Nerogar/OneTrainer/pull/193/commits/7c5f983bb728a8e5f6f6d62977727996d30b8e6c